scripts/create-release-branch.js

```diff
@@ -110,15 +110,22 @@ if (!isBetaRelease()) {
   shell.exec(`git fetch`)
 
```

Comment:
You could also push before and after the CL generation FWIW.  That'd probably make it simpler to let the CL fail and do the right thing in GHA

Reply:
We reviewed this one in a huddle - GHA is being pretty frustrating here and we're up against a clock for the release, so we're good to merge and come back for cleanup.